ethcore/private-tx/Cargo.toml

```diff
@@ -42,6 +42,7 @@ tiny-keccak = "1.4"
 trace = { path = "../trace" }
 transaction-pool = "2.0"
 url = "1"
+vm = { path = "../vm" }
```

Comment:
FWIW we may want to extract `vm::CreateContractAddress` and `ethcore::contract_address` in a separate crate.
parity (see https://github.com/paritytech/parity-common/issues/186).